GDJS/Runtime/CustomRuntimeObject.ts

```diff
@@ -363,6 +333,9 @@ namespace gdjs {
      * @returns the center X from the local origin (0;0).
      */
     getUnscaledCenterX(): float {
+      if (this._customCenter) {
```

Comment:
Was this a bug?

Reply:
It's a feature I forgot to add. Though, we had already done it for the shape painter version of the slider.